catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/controller/AbstractEventController.java

```diff
@@ -256,6 +256,7 @@ public void run() {
                 try {
                   TimeUnit.SECONDS.sleep(1);
                 } catch (InterruptedException e) {
+                  Thread.currentThread().interrupt();
```

Comment:
❗️ Same issue as before, the interruption must be propagated outside the loop using a finally clause